traccar/DOCS.md

```diff
@@ -105,16 +105,17 @@ By default, this add-on has disabled most of the GPS protocols. This has
 been done to reduce the number of open ports the add-on would create.
 
 By default, only the OsmAnd protocol (used by the Traccar Apps) and the API
-are enabled. If you want more protocols, you can do so, by adding entries
-to your `traccar.xml` file in the add-on configuration folder.
+are enabled. If you need to enable more protocols, you can do so by creating a copy of 
+[`traccar.xml`](https://github.com/hassio-addons/addon-traccar/blob/main/traccar/rootfs/etc/traccar/traccar.xml) 
+in your Home Assistant config directory and uncommenting the protocol your device uses.
```

Comment:
That is incorrect?

- One doesn't need to make a copy; the file is already there
- It doesn't reside in the Home Assistant config directory, but in the add-on configuration folder.